build.gradle.kts

```diff
@@ -133,6 +133,9 @@ subprojects {
             jvmTarget = "1.8"
             languageVersion = "1.6"
             apiVersion = "1.6"
+            if (JavaVersion.current().isJava9Compatible && project.name != "android") {
```

Comment:
Enabling this for android resulted in obscure errors:
```
> Task :examples:android:compileDebugKotlin FAILED
e: JDK_HOME path is not specified in compiler configuration
```

I'm guessing that it may use a different compiler version that doesn't support this argument.

Reply:
Is this something that's resolved (or simplified) with Gradle's [JVM Toolchain support](https://kotlinlang.org/docs/gradle-configure-project.html#gradle-java-toolchains-support)?

Reply:
I find that the toolchain support does more than I want in local testing - it is a bit too prescriptive.